codebase2/util/src/U/Util/Hash.hs

```diff
@@ -31,6 +32,9 @@ toBase32HexText = Base32Hex.toText . toBase32Hex
 fromBase32Hex :: Base32Hex -> Hash
 fromBase32Hex = Hash . B.Short.toShort . Base32Hex.toByteString
 
+unsafeFromBase32HexText :: Text -> Hash
```

Comment:
Note: this is slower than I'd like, due to the conversion from Text -> Bytestring -> ShortByteString